### PR TITLE
Add client test

### PR DESCRIPTION
**Pull request by robin-kunzler**

### CODE REVIEW

File: index_rs/src/index_rs/tests/tests.sh [hunk 1]
```diff
@@ -6,9 +6,11 @@
 // * adapt the canister IDs defined below to the ones that were deployed
 // * then run % ic-repl tests.sh
 
-import IndexCanister = "rrkah-fqaaa-aaaaa-aaaaq-cai";
+import IndexCanister = "q3fc5-haaaa-aaaaa-aaahq-cai";
+import ClientCanister = "qhbym-qaaaa-aaaaa-aaafq-cai";
```
Comment (fspreiss) [inline index_rs/src/index_rs/tests/tests.sh:10]: Now we have the client canister ID twice, but I guess that's fine.

**Review by fspreiss — APPROVED** [review]
Thanks, @robin-kunzler!

This is the output I get when running it (after updating the canister IDs):
```
Ping http://localhost:8000/...
Canister REPL
No history found
Current identity 7deel-f7olt-das4g-v62o2-ofbce-5i6sb-wd7n6-uyacm-5o3lz-6mkbi-hqe
2 : nat
                                                                                                                                                                            (3.38s)
opt record { client_canister_id = principal "q4eej-kyaaa-aaaaa-aaaha-cai" }
                                                                                                                                                                           (6.99ms)
null : null
                                                                                                                                                                            (4.69s)
null : null
```

Does that output indicate "success"?

File: index_rs/src/index_rs/tests/tests.sh (hunk 1 above)
Comment (robin-kunzler) [reply index_rs/src/index_rs/tests/tests.sh:10]: Hm yeah that's ugly, but I can't manage to extract a variable / constant for it. :-(

**Review by robin-kunzler — COMMENTED** [review]
@fspreiss : thanks for the review, please see my responses - I'll proceed to merge. 

> Does that output indicate "success"?

Yes it does, the output is not optimal, but it comes from the framework. If some assertion fails, we get an error here.

**Pull request merged**